src/main/resources/db/changelog/init/15062021_create_location_table.yml

```diff
@@ -16,15 +16,22 @@ databaseChangeLog:
                     primaryKeyName: pk_location
                     nullable: false
               - column:
-                  name: loc_country
-                  type: VARCHAR(50)
+                  name: country_code
+                  type: VARCHAR(255)
```

Comment:
length should be the same  as country_code in country lookup table

Reply:
Changed